src/Helper.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Helper.hpp"
+
+const std::string Helper::INPUT_FILES_DIRECTORY = "test_files";
+
+bool Helper::CreateTestFiles()
+{
+    //TODO: add Error handling
+    fs::create_directories(INPUT_FILES_DIRECTORY);
+    fs::path p1;
+    p1 /= INPUT_FILES_DIRECTORY;
+
+
+    // Create first test file
+    std::cout<<"File 'TestFile_1' is created" << std::endl;
```

Comment:
In a final version, this cout shouldn't be here. It can be good for debugging during development, but you shouldn't forget to remove it. Please add // TODO to keep it in mind.